doc/classes/EditorResourcePreviewGenerator.xml

```diff
@@ -9,41 +9,35 @@
 	<tutorials>
 	</tutorials>
 	<methods>
-		<method name="_can_generate_small_preview" qualifiers="virtual">
+		<method name="_can_generate_small_preview" qualifiers="virtual const">
 			<return type="bool" />
 			<description>
 				If this function returns [code]true[/code], the generator will call [method _generate] or [method _generate_from_path] for small previews as well.
 				By default, it returns [code]false[/code].
 			</description>
 		</method>
-		<method name="_generate" qualifiers="virtual">
+		<method name="_generate" qualifiers="virtual const">
 			<return type="Texture2D" />
-			<argument index="0" name="from" type="Resource" />
-			<argument index="1" name="size" type="Vector2" />
+			<argument index="0" name="resource" type="Resource" />
+			<argument index="1" name="size" type="Vector2i" />
 			<description>
-				Generate a preview from a given resource with the specified size. This must always be implemented.
-				Returning an empty texture is an OK way to fail and let another generator take care.
-				Care must be taken because this function is always called from a thread (not the main thread).
 			</description>
 		</method>
-		<method name="_generate_from_path" qualifiers="virtual">
+		<method name="_generate_from_path" qualifiers="virtual const">
 			<return type="Texture2D" />
 			<argument index="0" name="path" type="String" />
-			<argument index="1" name="size" type="Vector2" />
+			<argument index="1" name="size" type="Vector2i" />
 			<description>
-				Generate a preview directly from a path with the specified size. Implementing this is optional, as default code will load and call [method _generate].
-				Returning an empty texture is an OK way to fail and let another generator take care.
-				Care must be taken because this function is always called from a thread (not the main thread).
```

Comment:
This text was accidentally removed because the method signature changed and doctool lost it.

Reply:
oops, I've had that happen to me so many times :)